packages/xpath/test/xforms/now.test.ts

```diff
@@ -14,8 +14,6 @@ describe('#now()', () => {
 	// > including timezone offset (i.e. not normalized to UTC) as described
 	// > under the dateTime datatype.
 	it('should return a timestamp for this instant', () => {
-		// this check might fail if run at precisely midnight ;-)
```

Comment:
I think this comment wasn't related to the issue here but was a tongue and cheek remark about how it's possible for `new Date();` to be run milliseconds before midnight and then the `now()` function to be evaluated milliseconds after midnight in which case the test would fail even if the code is all correct. I do agree with removing the comment.